### ISSUE TITLE
🐞 [Bug]: Weird errors on staging, and dashboard.dev  logs

### ISSUE THREAD

**Issue by 0oM4R** [threefoldtech/tfgrid-sdk-ts#3342]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### which package/s did you face the problem with?

Dashboard

### What happened?

while opening the staging or deployed dashboard on **new incognito window**, you will see the errors on logs 

[Screencast from 2024-08-27 13-44-03.webm](https://github.com/user-attachments/assets/1a7de35f-bfab-47dc-8caa-3f85e0ec7868)

### which network/s did you face the problem on?

Dev

### version

2.5.0

### Relevant screenshots/screen records

[logs file](https://github.com/user-attachments/files/16762060/localhost-1724756165601.log)

### Relevant log output

```console
provided in previous section
```

**Comment by 0oM4R** [comment]
this is not related to lazy loading

**Comment by xmonader** [comment]
@samaradel investigate and let us know the root cause of this please

**Comment by samaradel** [comment]
update:
When `gridStore.client` is null initially, it can't access the `grid.twinId`, so it throws the error.

**Comment by 0oM4R** [comment]
I disagree with Samar, the `grid.twinId` is not accessed on any public pages, all weblets require the profile, and then the gridstore should be initialized 
Anyway, I tried to deploy a new version of the current staging version, which can be verified in http://185.206.122.44/#/

**Comment by zaelgohary** [comment]
Verified on the mentioned machine. https://github.com/threefoldtech/tfgrid-sdk-ts/issues/3342#issuecomment-3376510520
